tests/unit/ConversionNatureOfBusiness.spec.ts

```diff
@@ -0,0 +1,181 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { getVuexStore } from '@/store/'
+import { mount } from '@vue/test-utils'
+import ConversionNOB from '@/components/Conversion/ConversionNOB.vue'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+
+const businessInformation = {
+  legalType: 'SP',
+  identifier: '',
+  legalName: '',
+  foundingDate: '',
+  hasRestrictions: false,
+  naicsCode: '1234',
```

Comment:
the NAiCS code will always be six digits in the db so we should update this to use a 6 digit code

Reply:
The [schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/naics.json) states 5 to 9 digits. Does this change anything? (Other than this test which should be fixed.)

Reply:
Please use valid-length codes throughout this test suite.

Reply:
Thanks to both of you. I'll use 6 digits for the NAICS code for this test suite.